Homework_advanced_autobiography2/index.html

```diff
@@ -0,0 +1,73 @@
+﻿<!DOCTYPE  html>
+<html>
+ <head>
+  <title>Резюме</title>
+  <meta charset="utf-8"/>
+  <meta name="author"  content="Olkhovyk Boris"/>
+  <link rel="stylesheet" href="style.css"/>
+ </head>
+ <body>
+  <div class="header_div" >
+  <h1>Резюме</h1>
```

Comment:
this tag is only one on page, and this is block tag, why do you wrap it into `div`?